app/src/main/res/values-night/strings.xml

```diff
@@ -6,8 +6,42 @@
     <string name="health">Health</string>
     <string name="finance">Finance</string>
     <string name="lifeiswell">LifeIsWell</string>
+
+    <string name="mentalpage">Mental Wellness</string>
+    <string name="title_activity_meditation_page">MeditationPageActivity</string>
+    <string name="meditations">Meditations</string>
+
+    <string name="gratitudeJournal">Gratitude Journal</string>
+    <string name="breathe">Breathing Exercise</string>
+    <string name="lists">TO DOs</string>
+
+    <!-- Strings used for fragments for navigation -->
+    <string name="first_fragment_label">First Fragment</string>
+    <string name="second_fragment_label">Second Fragment</string>
+    <string name="next">Next</string>
+    <string name="previous">Previous</string>
+
+    <string name="hello_first_fragment">Hello first fragment</string>
+    <string name="hello_second_fragment">Hello second fragment. Arg: %1$s</string>
+    <string name="title_activity_meditation_page_main">MeditationPageMain</string>
+    <string name="title_home">Home</string>
+    <string name="title_dashboard">Dashboard</string>
+    <string name="title_notifications">Notifications</string>
+    <string name="title_activity_breathing_exercixe">BreathingExercixe</string>
+    <string name="title_activity_breathing_exercise_mental_page">BreathingExerciseMentalPage</string>
+    <string name="defaultNavHost">nav_host</string>
+    <string name="navGraph">nav_graph</string>
+    <string name="nameofuser">Name</string>
+    <string name="account">Account 1</string>
+    <string name="new_account">New Account</string>
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```

Comment:
Just making note for me and anyone else that this lil section can be removed at any point along the road.